plugins/woocommerce-beta-tester/src/tools/commands/set-woocom-base-url.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import { TextControl } from '@wordpress/components';
+import { useDispatch, useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { STORE_KEY } from '../data/constants';
+
+export const UPDATE_WOOCOM_BASE_URL_ACTION_NAME = 'updateWoocomBaseUrl';
+
+export const SetWoocomBaseUrl = () => {
+	const url = useSelect(
+		( select ) => select( STORE_KEY ).getWoocomBaseUrl(),
+		[]
+	);
+	const { updateCommandParams } = useDispatch( STORE_KEY );
+
+	function onUpdate( newUrl ) {
+		updateCommandParams( UPDATE_WOOCOM_BASE_URL_ACTION_NAME, {
+			url: newUrl,
+		} );
+	}
+
+	return (
+		<div className="woocom-base-url-control">
+			{ url === undefined ? (
```

Comment:
Should we change to falsy check `! url` here since `url` is default to `''`?

Reply:
Yeah, good catch! Fixed in 3c38f9f06786a15bda75926c64f5da320c2fdcde.

Reply:
I found this would cause an issue so I changed the default value to https://woocommerce.com/ instead in https://github.com/woocommerce/woocommerce/pull/51537/commits/0619c6fa7f8d09f1a58e76c92a569559b19b2480.